src/App.js

```diff
@@ -15,6 +16,8 @@ const App = () => {
         <Route path="/" element={<LandingPage />} />
         <Route path="/login" element={<Login />} />
         <Route path="/register" element={<Register />} />
+        <Route path="/" element={<Home />} />
```

Comment:
LandingPage and Home component are same, please remove Home

Reply:
Done